eemeter/co2/clients.py

```diff
@@ -0,0 +1,120 @@
+from io import BytesIO
+import logging
+from datetime import datetime
+
+import pandas as pd
+
+import requests
+import zipfile
+import xlrd
+
+logger = logging.getLogger(__name__)
+
+AVERT_NAMES = {
+    'CA': ['California', 'california'],
+    'EMW': ['Great Lakes - Mid-Atlantic', 'great_lakes_-_mid-atlantic'],
+    'LMW': ['Lower Midwest', 'lower_midwest'],
+    'NE': ['Northeast', 'northeast'],
+    'NW': ['Northwest', 'northwest'],
+    'RM': ['Rocky Mountains', 'rocky_mountains'],
+    'SE': ['Southeast', 'southeast'],
+    'SW': ['Southwest', 'southwest'],
+    'TX': ['Texas', 'texas'],
+    'UMW': ['Upper Midwest', 'upper_midwest']
+}
+
+
+class AVERTClient(object):
```

Comment:
These clients are a bit trickier to test since they rely on an external service. In writing the tests for the others parts you may need to spoof this client so we aren't being too rude.